pod/video/templates/videos/video-info.html

```diff
@@ -406,13 +406,12 @@ <h2 class="modal-title h3" id="shareLabel">
                 <label for="txtpartage">{% trans 'Use this link to share the video:' %}</label>
                 <input class="form-control" type="text" name="txtpartage" id="txtpartage" value="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% if enrichment%}{% url 'enrichment:video_enrichment' slug=video.slug %}{% else %}{% url 'video:video' slug=video.slug %}{% endif %}{% if video.is_draft == True %}{{ video.get_hashkey }}/{% endif %}" readonly>
               </div>
-              <div class="form-group">
-                <label>{% trans 'QR code for this link:' %}&nbsp;
-                  <a href="#" class="btn pod-btn pod-btn-social m-1" title="{% trans 'Warning, it use google API' %}" role="button" data-bs-toggle="collapse" data-bs-target="#qrcode" aria-expanded="false" aria-controls="qrcode">
-                    <i class="bi bi-eye"></i>
-                  </a>
-                </label>
-                <img src="" data-src="//chart.apis.google.com/chart?cht=qr&amp;chs=200x200&amp;chl={% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% if enrichment%}{% url 'enrichment:video_enrichment' slug=video.slug %}{% else %}{% url 'video:video' slug=video.slug %}{% endif %}{% if video.is_draft == True %}{{ video.get_hashkey }}/{% endif %}" alt="{% trans 'QR code' %}" id="qrcode" loading="lazy" class="collapse">
+              <div class="form-group m-2">
+                <label>{% trans 'QR code for this link:' %}&nbsp;</label>
+                  <a href="#" class="btn btn-primary m-2" title="{% trans 'Warning, it use google API' %}" role="button" data-bs-toggle="collapse" data-bs-target="#qrcode" aria-expanded="false" aria-controls="qrcode">{% trans 'Voir le QR Code' %}</a>
+                  <button id="btnDowloadQrCode" type="submit" data-slug={{ video.slug }} class="btn btn-primary" title="{% trans 'Warning, it use google API' %}">{% trans 'Download' %}</button>
+                  <img src="" data-src="//chart.apis.google.com/chart?cht=qr&amp;chs=200x200&amp;chl={% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% if enrichment%}{% url 'enrichment:video_enrichment' slug=video.slug %}{% else %}{% url 'video:video' slug=video.slug %}{% endif %}{% if video.is_draft == True %}{{ video.get_hashkey }}/{% endif %}"
+                     alt="{% trans 'QR code' %}" id="qrcode" loading="lazy" class="collapse m-2">
```

Comment:
En balise alt, je mettrais plutôt un truc du style : `alt="{% blocktrans with video_title=video.title %}QR code to access the video “{{ video_title }}”{% endblocktrans %}`